src/isar/state_machine/states/stop_step.py

```diff
@@ -20,6 +20,9 @@ def __init__(self, state_machine: "StateMachine"):
         self.state_machine: "StateMachine" = state_machine
         self.logger = logging.getLogger("state_machine")
         self.stop_thread = None
+        self._count_number_retries = 0
+        self._has_failed_current = False
+        self.has_failed_list: List[bool] = []
```

Comment:
Do we need for this list for anything other than the tests?

Reply:
Kun i test foreløpig.

Reply:
Then we should try to adjust the tests so that it is not needed